enhancements/installer/ibm-cloud-for-power-vs-ipi.md

```diff
@@ -0,0 +1,231 @@
+---
+title: ibm-cloud-ipi
+authors:
+  - "@jaypoulz"
+reviewers:
+  - "@crawford"
+  - "@andymcc"
+  - "@Prashanth684"
+approvers:
+  - TBD
+creation-date: 2021-04-16
+last-updated: 2021-04-16
+status: implementable
+---
+
+# ibm-cloud-provider-for-power-vs-platform
+
+## Release Signoff Checklist
+
+- [x] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Operational readiness criteria is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift-docs](https://github.com/openshift/openshift-docs/)
+
+## Summary
+
+This document describes how the [IBM Cloud][ibm-cloud-website] can be leveraged to provide infrastructure to deploy
+OpenShift on the [Power Virtual Server][power-vs-website] offering. Power Virtual Server, or Power VS, is an IBM Cloud
+offering that aims to provide customers interested in running secure and flexible workloads on IBM Power (ppc64le)
+hardware without having to develop and maintain on premise datacenters. The goal of this enhancement would be to
+leverage the IBM Cloud APIs to deploy an OpenShift cluster on this cloud instance hardware. In addition, the APIs would
+be leveraged for cluster resizing operations post installation.
+
+## Motivation
+
+- Current deployments of OpenShift on Power use UPI, with the exception of the development only configuration of IPI on
+libvirt.
+- Currently, the Multi-Arch team is stuck using a development configuration (IPI on libvirt) to cover our CI workloads
+for OpenShift on ppc64le verification. This would help diversify our CI effort.
+- A non-x86 IPI installer would serve as a first step towards improving the ease of deployments on non-x86 hadware.
+- Power hardware is especially targeted to AI and secure workloads, so building out cloud capabilities will help expand
+our workload footprint.
+- IPI is generally easier to install and set up quickly for demos and proof of concepts, when can help lower the barrier
+of entry for power optimized workloads.
+
+### Goals
+
+- Provide a way to install OpenShift on Power VS infrastructure using the OpenShift installer with IPI provisioned
+against IBM Cloud APIs.
+- Implement the cluster-api provider to provide scaling and managing the cluster nodes post install.
+
+### Non-Goals
+
+- UPI support. We've already documented how to install on pp64le hardware using UPI.
+
+## Proposal
+
+In order to leverage the Power VS platform, we will first need to implement an IBM Cloud Provider. This provider will
```

Comment:
probably would be good to call out that a terraform-provider would be used for provisioning control plane and the cluster-api-provider for the computes.